consensus/consensus_v2.go

```diff
@@ -761,6 +768,29 @@ func (consensus *Consensus) finalizeCommits() {
 func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	consensus.getLogger().Debug().Msg("[OnCommitted] Receive committed message")
 
+	// TODO: this is temp hack for update new node's committee information; remove it after staking and resharding finished
+	if consensus.mode.Mode() == Listening {
+		recvMsg, err := ParsePbftMessage(msg)
+		if err != nil {
+			consensus.getLogger().Warn().Msg("[OnCommitted] unable to parse msg")
```

Comment:
error or warning?  If error, please print out err.

Reply:
warning. it's quite possible we received invalid message from other peers.